### ISSUE TITLE
Superfluous variable in textual definition of ‘first instant of’

### ISSUE THREAD

**Issue by gregfowlerphd** [BFO-ontology/BFO-2020#98]
**Describe the bug**

The definition reads:

> t first instant of t' =Def t is a temporal instant & t' is a temporal region t' & t precedes all temporal parts of t' other than t

Take a look at the second conjunct. The second instance of t’ there (after 'region') seems to be a typo.

**Expected behavior**

The relevant instance of t' should be deleted.

**Comment by alanruttenberg** [comment]
yes. Fixed in source. Needs to be fixed in owl files unless synced.

**Comment by gregfowlerphd** [comment]
@alanruttenberg: Yeah, that's odd. It looks like it must have been introduced when it was decided to move the specifications of t as a temporal instant and t' as a temporal region to the right-hand side of the "=Def". Notice that "temporal region t' " appears in the source. Presumably it was just copied-and-pasted over to the right-hand side and then "t' is a" was inserted in front of it.
